interpreter/cling/lib/Utils/AST.cpp

```diff
@@ -114,8 +114,7 @@ namespace utils {
 
     Expr* result = nullptr;
     if (CompoundStmt* CS = dyn_cast<CompoundStmt>(FD->getBody())) {
-      ArrayRef<Stmt*> Stmts
-        = llvm::makeArrayRef(CS->body_begin(), CS->size());
+      ArrayRef<Stmt*> Stmts = llvm::ArrayRef(CS->body_begin(), CS->size());
```

Comment:
Could just be
```suggestion
      ArrayRef<Stmt*> Stmts(CS->body_begin(), CS->size());
```
I guess?